x-pack/plugins/security_solution/common/endpoint/index_data.ts

```diff
@@ -103,7 +105,7 @@ export async function indexHostsAndAlerts(
   }
 
   for (let i = 0; i < numHosts; i++) {
-    const generator = new DocGenerator(random);
+    const generator = new DocGenerator(random, undefined);
```

Comment:
Optional: you don't need `undefined` as the second argument since the argument is optional